src/types/profile.js

```diff
@@ -304,10 +304,10 @@ export type Counter = {|
   description: string,
   pid: Pid,
   mainThreadIndex: ThreadIndex,
-  sampleGroups: {|
+  sampleGroups: Array<{|
```

Comment:
same as above, can you try to use a `$ReadOnlyArray` here? Although I'm not sure we can for this one, as we mutate it when we create it.

Reply:
It looks like it's working :)